frontend/src/app/[locale]/formPrototype/[applicationId]/[formId]/page.tsx

```diff
@@ -77,9 +123,11 @@ async function FormPage({ params }: formPageProps) {
         ).
       </p>
       <ApplyForm
+        rawFormData={application_response}
```

Comment:
does the `application_response` / `rawFormData` represent a user's saved application state? If so how can we make that more clear using the variable naming here?

Reply:
Yes, this is the saved state from the server that hasn't yet been loaded into the form. Do you have one in mind? It's not `formData` which is already accounted and "initial" or `formDataProps` doesn't seem more clear to me.

Reply:
something like `savedApplicationData`? `userSavedApplication`?